lib/__tests__/test-stack.ts

```diff
@@ -38,6 +39,7 @@ export class TestStack extends Stack {
         DELIVLIB_ENV_TEST: 'MAGIC_1924',
       },
       dryRun: true,
+      buildImage: LinuxBuildImage.fromDockerRegistry('public.ecr.aws/s9s5g8n5/superchain:1-buster-slim-node18'),
```

Comment:
Once we have the alias `jsii` approved, we can replace this randomly generated name with it.